### PR TITLE
dope ass home screen

### PR DESCRIPTION
**Pull request by zteching**
update

### CODE REVIEW

**Review by leopolicastro — APPROVED** [review]

**Comment by naaltunian** [comment]
let's avoid commit messages like "dope ass". Doesn't look professional. Also there's a conflict in Home.jsx.

File: package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
     "mongodb": "^3.5.8",
     "mongoose": "^5.9.16",
     "multer": "^1.4.2",
+    "node-sass": "^4.14.1",
```
Comment (Stommy123) [inline package.json:22]: I don't see sass being used anywhere in this codebase. Could we please remove this dependency?

**Review by Stommy123 — CHANGES_REQUESTED** [review]
Looks good! Small comment about an unused dependency

**Comment by naaltunian** [comment]
lots of conflicts :/

**Pull request closed without merging**